Headers/AppKit/NSComboBox.h

```diff
@@ -106,13 +106,19 @@
 #endif
 @end
 
+@protocol NSComboBoxDataSource <NSObject>
```

Comment:
I don't see the point of having an empty protocol along a defined interface. Either replace the interface with the protocol completely or do something like 
#if OS_API_VERSION(MAC_OS_X_VERSION_10_6, GS_API_LATEST)
@protocol NSComboBoxDataSource <NSObject>
#else
@interface NSObject (NSComboBoxDataSource)
#endif

And the same is true for NSComboBoxDelegate

Reply:
The point would be to allow code which uses those protocols to compile, at least that was my initial thought.  The effect is basically the same, as all methods are @optional in both protocols.   I will change it in the way that you suggest (your second suggestion).  Thx... GC

Reply:
Fix committed.  I'm resolving this.